postcss.config.js

```diff
@@ -5,5 +5,12 @@ module.exports = ({ env }) => ({
     require('postcss-preset-env')({ stage: 1 }), // https://preset-env.cssdb.org/features#stage-1
     // Other plugins...
     env === 'production' ? require('cssnano')({ preset: 'default' }) : null, // How to use conditional plugins/options for prod
```

Comment:
Thought: I don't _think_ it'll have unitended outcomes, but do you think it's worth running `cssnano` as the last step in the postcss build? A) it is conditional based on env, and B) it's another transformation to the CSS file that could potentially manipulate the raw CSS and interfere with purging?

Seems to be behaving fine either way around. Just a hunch.

Reply:
Like you say it does seem to be working either way but having `cssnano` as the last transformation makes sense.